tools/windows/DatadogAgentInstaller/CustomActions/PrerequisitesCustomActions.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Security.Principal;
+using System.Windows.Forms;
+using Datadog.CustomActions.Interfaces;
+using Microsoft.Deployment.WindowsInstaller;
+
+namespace Datadog.CustomActions
+{
+    public class PrerequisitesCustomActions
+    {
+        const string Error = "The Datadog Agent installer must be run by a user that is a member of the Administrator group.";
+
+        [CustomAction]
+        public static ActionResult EnsureAdminCaller(Session session)
+        {
+            if (!new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator))
```

Comment:
Is this invoked before a jump to system process? ... But after UAC prompt (on normal installation)?

Reply:
yep, thank you, blind